app/assets/json/translations/react/en.json

```diff
@@ -1007,14 +1008,14 @@
   "lottery.enterLotteryNumber": "Enter Your Lottery Number",
   "lottery.generalPool": "General Pool",
   "lottery.housingPreferences": "Housing Preferences",
+  "lottery.lottery": "Lottery",
```

Comment:
I've noticed there's a lottery.lottery and lottery key that have the same value. I wonder if there's others key/values this state- could potentially have update one of the values and a future developer chooses the wrong one. Could be worthwhile some day to search for duplicates?